src/classes/RLLP_OppRollup.cls

```diff
@@ -80,109 +91,199 @@ public without sharing class RLLP_OppRollup {
+            }
         }
-        
+
         //contacts needs to be done async to make sure the OCR is in place
         if (!contactRequerySet.isEmpty()){
-        	if (system.isFuture() || system.isBatch())
-                RLLP_OppRollup.rollupContactsandHouseholdsForTrigger(contactRequerySet);                        
-            else        	
-                RLLP_OppRollup.rollupContactsandHouseholdsForTriggerFuture(contactRequerySet);                        
+            if (system.isFuture() || system.isBatch())
+                RLLP_OppRollup.rollupContactsandHouseholdsForTrigger(contactRequerySet);
+            else
+                RLLP_OppRollup.rollupContactsandHouseholdsForTriggerFuture(contactRequerySet);
         }
     }
-    
+
     /*******************************************************************************************************
     * @description Future wrapper method for updating contacts and households from a set of opportunity IDs.
     * @param OppIds A set of opportunity ids.
     * @return void This method does updates directly and returns nothing.
     ********************************************************************************************************/
-    @future 
+    @future
     public static void rollupContactsandHouseholdsForTriggerFuture(set<id> OppIds) {
         rollupContactsandHouseholdsForTrigger(OppIds);
     }
 
     /*******************************************************************************************************
     * @description Updating contacts and households rollups from a set of opportunity IDs.
-    * @param OppIds A set of opportunity ids.
+    * @param oppIds A set of opportunity ids.
     * @return void This method does updates directly and returns nothing.
     ********************************************************************************************************/
-    public static void rollupContactsandHouseholdsForTrigger(set<id> OppIds){
+    public static void rollupContactsandHouseholdsForTrigger(Set<Id> oppIds){
         //we need this to make sure there's a proper OCR when Contact Opps are inserted
-        
-        map<id, Contact> cmap = new map<id, Contact>();
-        map<id, npo02__Household__c> hhmap = new map<id, npo02__Household__c>();
-        
         // use the contact roles to find the contacts and households
-        RLLP_OppRollup_UTIL oppRollupUtil = new RLLP_OppRollup_UTIL();
-        for (OpportunityContactRole r : Database.query(oppRollupUtil.buildOpportunityContactRoleQuery() +
-                                            ' where Opportunity.Id IN :OppIds and isPrimary = true ALL ROWS ')){   
-                    
-            cmap.put(r.ContactId, r.Contact);
-                
-            if (r.Contact.npo02__Household__c != null){
-                hhmap.put(r.Contact.npo02__Household__c, r.Contact.npo02__Household__r);
+        String query = new RLLP_OppRollup_UTIL().buildOpportunityContactRoleQuery() +
```

Comment:
I'm not necessarily sure this is going to matter 99.9% of the time, but it's worth noting that there is a difference in how legacy rollups identify "Primary" for the Contact Hard Credit rollups:
* Legacy Rollups use the `OpportunityContactRole.IsPrimary` field to determine which Contact gets hard credit. If there are dupe OCR"s with IsPrimary=True (which is technically possible) this could lead to two Contacts getting Hard Credit for the same OCR. There's a fixup job in NPSP Settings to address the dupe OCR thing.
* Customizable Rollups use the `Opportunity.Primary_Contact__c` field to determine which Contacts get Hard Credit.

So, for this job to determine which Contacts will get Hard Credit (and thus need a recalc) by querying the OCR for specific Opportunities could - though extremely unlikely - lead to a situation where the actual Primary Contact (the one specified by `Opportunity.Primary_Contact__c`) is not recalculated. This won't mis-calculate amounts because Customizable Rollups will still use the correct logic to make the Hard Credit determination. However, if for some reason the Contact identified by `Opportunity.Primary_Contact__c` does not have a matching OpportunityContactRole record with IsPrimary set to true, then that PrimaryContact would have their values recalculated by this job. Again, this is extremely unlikely and even if it does happen the Contact would get recalculated in the nightly job.

Reply:
This change was strictly formatting and should not have changed any logic. Anything related to household rollups should be handled in a separate work item.